src/Modules/Meetings/Tests/IntegrationTests/SeedWork/TestBase.cs

```diff
@@ -104,6 +104,7 @@ private static async Task ClearDatabase(IDbConnection connection)
                                "DELETE FROM [meetings].[Meetings] " +
                                "DELETE FROM [meetings].[MeetingWaitlistMembers] " +
                                "DELETE FROM [meetings].[MeetingComments] " +
+                               "DELETE FROM [meetings].[MeetingCommentingConfigurations] " +
```

Comment:
This deletion should be **before** deletion of Meetings table because of FK.

Reply:
Thanks, fixed.